modules/ROOT/pages/partner-manager-architecture.adoc

```diff
@@ -1,2 +1,63 @@
 = Anypoint Partner Manager Architecture
 
+With Anypoint Partner Manager, you can deploy B2B Integration message flows with your partners, either to your own hosted Mule runtimes or to MuleSoft hosted CloudHub iPaaS platform.
+
+The functional features and user experience in configuring B2B Integrations with Partner Manager are deployment-target agnostic; however, there are differences in the underlying architecture when implementing the functionality in the runtime.
+
+When you deploy a configured message flow, Partner Manager automatically translates the configurations used in the message flow and deploys and updates a set of applications necessary to orchestrate end-to-end B2B processing to the Mule runtime. These applications include:
+
+* Receive endpoint services
+* Document processing service
+* Replication service
+* Send endpoint services
+
+image::partner-manager-high-level-arch.png[Anypoint Partner Manager Architecture]
+
+[%header%autowidth.spread]
+|===
+|Callout |Name |Description
+|image:pm-arch-diagram-1.png[]
+|Partner Manager
+|MuleSoft hosted cloud control plane allowing users to configure trading partner profiles, build and deploy B2B integration message flows, and monitor B2B transactions activity.
+
+|image:pm-arch-diagram-2.png[]
+|Receive Endpoint Services
+|Service to receive inbound B2B or EDI messages from partners transport protocols such as AS2 and SFTP or outbound application messages from your organization’s backend systems via HTTP, HTTPS, or SFTP.
+
+|image:pm-arch-diagram-3.png[]
+|Document Processing Service
+|Core B2B processing engine responsible for parsing, validating and translating received B2B messages based on configurations and mapping in Partner Manager.
+
+|image:pm-arch-diagram-4.png[]
+|API Replication Service
+|Tracking service responsible for updating transmissions and transaction activities to the control plane and to the payload storage API.
+
+|image:pm-arch-diagram-5.png[]
+|Send Endpoint Services
+|Service to deliver translated inbound application messages to internal applications such as ERP, billing, Order Management via HTTP, HTTPS or SFTP; or outbound B2B/EDI payloads to your trading partners via AS2, HTTP, HTTPS or SFTP.
+
+|image:pm-arch-diagram-6.png[]
+|Payload Storage API
+|API to store B2B payload data in your own storage solution.
```

Comment:
no period

The ideal is that list item descriptions are parallel (either all fragments or all periods, without and with periods, respectively). If your descriptions aren't parallel, and at least one includes a complete sentence, then all must end with periods. But it's best to use parallel formats.

Reply:
Got it!